pkg/announcements/types.go

```diff
@@ -156,6 +156,17 @@ const (
 
 	// EgressUpdated is the type of announcement emitted when we observe an update to egress.policy.openservicemesh.io
 	EgressUpdated AnnouncementType = "egress-updated"
+
+	// ---
+
+	// MultiClusterServiceAdded is the type of announcement emitted when we observe an addition of a multiclusterservice.config.openservicemesh.io
+	MultiClusterServiceAdded AnnouncementType = "multiclusterservice-added"
+
+	// MultiClusterServiceDeleted is the type of announcement emitted when we observe an deletion of a multiclusterservice.config.openservicemesh.io
+	MultiClusterServiceDeleted AnnouncementType = "multiclusterservice-deleted"
+
+	// MultiClusterServiceUpdated is the type of announcement emitted when we observe an update of a multiclusterservice.config.openservicemesh.io
+	MultiClusterServiceUpdated AnnouncementType = "multiclusterservice-updated"
```

Comment:
You will need to add this to this file as well : https://github.com/openservicemesh/osm/blob/113bfc328b857272dc149066d162fc950f0b584f/pkg/catalog/dispatcher.go#L43